detectors/gcp/detector.go

```diff
@@ -36,6 +36,7 @@ const (
 	GKE
 	GCE
 	CloudRun
+	CloudRunJobs
```

Comment:
I don't think we need a new cloud run jobs constant here, since the detector will use the same functions (e.g. FaasName) to add resource attributes.

Reply:
Can you instead update the onCloudRun to return true on cloudRunJobs as well?